pkgs/development/libraries/libbfio/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, fetchurl
+, fetchpatch
+, stdenv
+, gettext
+, libtool
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  version = "20221025";
+  pname = "libbfio";
+
+  src = fetchurl {
+    url = "https://github.com/libyal/libbfio/releases/download/${version}/libbfio-alpha-${version}.tar.gz";
+    hash = "sha256-GBS9oVZxdvo4Sv8DaZU0t97aN1oHktkvAsPpemLlOyc=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ gettext libtool ];
+
+  meta = {
+    description = "Library to provide basic file input/output abstraction";
+    homepage = "https://github.com/libyal/libbfio";
+    license = lib.licenses.lgpl3;
+    maintainers = with lib.maintainers; [ d3vil0p3r ];
+    platforms = lib.platforms.unix;
```

According to upstream docs, this package supports windows as well.

```suggestion
    platforms = with lib.platforms; unix ++ windows;
```